lib/active_model/serializer/field.rb

```diff
@@ -36,6 +41,7 @@ def excluded?(serializer)
           false
         end
       end
+      add_method_tracer :excluded?, 'ActiveModel::Serializer::Field/excluded?'
```

Comment:
These should be prefixed with `Custom/`: http://www.rubydoc.info/github/newrelic/rpm/NewRelic/Agent/MethodTracer/ClassMethods#add_method_tracer-instance_method